### DISCUSSION TITLE
Elsa core 3.0

### DISCUSSION THREAD

**Discussion by chandusekhar** [elsa-workflows/elsa-core#3683] [Q&A]
What are the timelines for Elsa core 3.0 release

**Comment by sfmskywalker** [comment]
It looks like an RC might happen sometime towards the end of March. We're currently working on the workflow journal and input/output support of activities, allowing you to create workflows in the designer that can then be used as composite activities in other workflows. Once that is done, we need a bit of time to complete the initial version of the documentation.
